lms/djangoapps/grades/new/course_grade.py

```diff
@@ -24,8 +26,35 @@ class CourseGrade(object):
     def __init__(self, student, course, course_structure):
         self.student = student
         self.course = course
+        self.course_version = getattr(course, 'course_version', None)
+        self.course_edited_timestamp = getattr(course, 'subtree_edited_on', None)
         self.course_structure = course_structure
         self.chapter_grades = []
+        self.persisted_grade = None
```

Comment:
Yeah, @nasthagiri had the same comment. I think we're going to move the grading policy hash comparison down into `init_from_model`, that will eliminate the need for the `grading_policy_hash` property on `CourseGrade`. Then `_percent` and `_letter_grade` can just be individual fields.